go/exorelay/exorelay_test.go

```diff
@@ -16,26 +16,26 @@ import (
 	"github.com/Originate/exocom/go/structs"
 )
 
+func initializeExocom() *exocomMock.ExoComMock {
+	exocom := exocomMock.New()
+	go func() {
+		err := exocom.Listen(4100)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}()
+	defer exocom.Close()
+	return exocom
+}
```

Comment:
I have yet to see a platform that can tolerate opening and closing a lot of sockets on the same port. Too much heavyweight OS stuff underneath I guess.